components/match2/wikis/trackmania/match_group_input_custom.lua

```diff
@@ -154,6 +154,17 @@ function matchFunctions.getVodStuff(match)
 	return match
 end
 
+function matchFunctions.isFeatured(match)
+	if
+		tonumber(match.liquipediatier or '') == 1
+		or tonumber(match.liquipediatier or '') == 2
+		or not String.isEmpty(Variables.varDefault('match_featured_override'))
+	then
+		return true
+	end
+	return false
```

Comment:
```suggestion
	return tonumber(match.liquipediatier or '') == 1
		or tonumber(match.liquipediatier or '') == 2
		or not String.isEmpty(Variables.varDefault('match_featured_override'))
```

Reply:
`match_featured_override` gets never set as a variable anywhere so the `or not String.isEmpty(Variables.varDefault('match_featured_override'))` check seems useless

additionally the `or ''` in the `tonumber` is not needed